Android/app/src/main/java/app/intra/socks/OverrideSocksHandler.java

```diff
@@ -166,6 +168,8 @@ public void doConnect(Session session, CommandMessage commandMessage) throws IOE
       // - VALUE : total transfer over the lifetime of a socket
       // - PORT : TCP port number (i.e. protocol type)
       // - DURATION: socket lifetime in seconds
+      // - LATENCY: TCP handshake latency in milliseconds
```

Comment:
Maybe LATENCY -> TCP_HANDSHAKE_MS?
Latency is not descriptive. And it would be useful to have the unit, specially given DURATION uses seconds.

Are these strings supposed to be short?

Reply:
Done.  The strings should be short, but hopefully they'll be compressed so it doesn't matter.